admin/bin/qserv-configure.py

```diff
@@ -96,32 +100,53 @@ def __init__(self):
+                            help='Run initialization and configuration'
                             )
         # Defining option of each configuration step
-        for step_name in configure.STEP_LIST:
-            parser.add_argument(
-                "-{0}".format(configure.STEP_ABBR[step_name]),
+        init_group = parser.add_argument_group('Initialization', 'Creation of QSERV_RUN_DIR and QSERV_DATA_DIR')
+        config_group = parser.add_argument_group('Configuration steps', 'General configuration steps')
+        nodb_component_group = parser.add_argument_group('Components configuration', 'Configuration of external components')
+        db_component_group = parser.add_argument_group('Database components configuration', 'Configuration of external components '
+                                                       'impacting data,\nlaunched if and only if QSERV_DATA_DIR is empty')
+        for step_name in configure.ALL_STEPS:
+            if step_name in configure.INIT:
+                group = init_group
+            elif step_name in configure.DB_COMPONENTS:
+                group = db_component_group
+            elif step_name in configure.NODB_COMPONENTS:
+                group = nodb_component_group
+            else:
+                group = config_group
+            group.add_argument(
+                "-{0}".format(configure.ALL_STEPS_SHORT[step_name]),
                 "--{0}".format(step_name),
                 dest="step_list",
                 action='append_const',
                 const=step_name,
-                help=configure.STEP_DOC[step_name]
+                help=configure.ALL_STEPS_DOC[step_name]
             )
 
         # forcing options which may ask user confirmation
         parser.add_argument("-f", "--force", dest="force", action='store_true',
                             default=False,
-                            help="forcing removal of existing execution data"
+                            help="Answer yes to all questions, use with care. Default: %(default)s"
                             )
 
         # run dir, all mutable data related to a qserv running instance are
         # located here
         parser.add_argument("-R", "--qserv-run-dir", dest="qserv_run_dir",
                             default=default_qserv_run_dir,
-                            help="full path to qserv_run_dir"
+                            help="Absolute path to qserv_run_dir. Default: %(default)s"
+                            )
+
+        # run dir, all mutable data related to a qserv running instance are
+        # located here
+        init_group.add_argument("-D", "--qserv-data-dir", dest="qserv_data_dir",
+                            default=None,
+                            help="Absolute path to directory containing Qserv data, default to QSERV_RUN_DIR/var/lib. "
+                                 "IMPORTANT: Set QSERV_DATA_DIR outside of QSERV_RUN_DIR to protect data when configuring Qserv."
```

Comment:
This (and few string literals above) exceeds 110-character limit which is dictated by our coding standards.